joomla/joomla-cms
# [ACL] [new/edit category] Remove categoryedit field check of - No parent - option

### Summary of Changes

The categoryedit field (which actually is the category parent select box) is making an ACL check to the "- No parent -" option to decide whether to show the field or not. 
![image](https://cloud.githubusercontent.com/assets/9630530/20076556/bddd5a7e-a530-11e6-90e9-706a0f8be726.png)

This doesn't make much sense. This PR correct that.

Note: it will also be a bit faster since now the ACL check is the last of the condition.

### Testing Instructions

Mainbly code review, but you can also:
1. Apply patch
2. Check category edit field work as usual

### Documentation Changes Required

none.
